migration/migrator/migrations/course/20221028222200_poll_response_migration_v2.py

```diff
@@ -0,0 +1,45 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("""
+    ALTER TABLE poll_responses DROP CONSTRAINT IF EXISTS poll_responses_id_seq;
```

Comment:
May want to add indexes into the table on parts such as student_id in poll_responses (used in findAllByStudentIDWithAllOptions)

Reply:
Given that a large number of insert operations will be taking place as a class submits responses to a poll, and that there's already an index on the primary key which identifies all of the responses associated with a given poll, I'm somewhat wary of adding additional indices to the responses table.  Thoughts?

I have no hard data to reference unfortunately, so I'm definitely open to counterarguments on this matter.  Perhaps we could get this PR merged first, and then play around with the indexing sometime later down the road.